File: 황승민:2주차 과제/Q03.c
/* 
3. num1이라는 숫자를 입력받아 50보다 작으면 num2는 0, 크거나 같으면 100을 출력하는 프로그램을 만들어보자
(조건 3항 연산자 이용) 
*/

#include <stdio.h>

int main(void) {

    int num1 = 0, num2 = 0;

    scanf("%d", &num1);
    num2 = num1 < 50 ? 0 : 100;

    printf("%d\n", num2);   

    return 0;
}

File: EunSeo/javaWeek3/Q03.java
package javaWeek3;

import java.util.Scanner;

public class Q03 {

    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);

        int[] nums1 = new int[5];
        int[] nums2 = new int[5];

        int num1 = scanner.nextInt();
        int num2 = scanner.nextInt();
        int sum = 0;

        for (int i = 4 ; i >= 0 ; i--) {
            nums1[i] = num1 / (int) Math.pow(10, i);
            if (nums1[i] > 9) {
                nums1[i] = 0;
            }
            num1 %= (int) Math.pow(10, i);
            nums2[i] = num2 / (int) Math.pow(10, i);
            if (nums2[i] > 9) {
                nums2[i] = 0;
            }
            num2 %= (int) Math.pow(10, i);
        }

        for (int i = 0 ; i < 5 ; i++) {
            for (int j = 0 ; j < 5 ; j++) {
                sum += nums1[i] * nums2[j];
            }
        }

        System.out.println(sum);
    }
}

